Update `@unocha/hpc-repo-tools` to `v7.1.0`

Pl217 opened the pull request:
ESLint and Typescript are peer dependencies and need to be updated at the same time.

Pl217 committed:
Update `@unocha/hpc-repo-tools` to `v7.1.0`

ESLint and Typescript are peer dependencies
and need to be updated at the same time

unocha-hpc reviewed (approved):
Checks have passed and this pull request is ready for manual review

Pl217 committed:
HPC-10213: Remove usage of legacy Dockerfile ENV syntax

The syntax for `ENV` instructions that doesn't use an
assignment operator is being phased out.
https://github.com/docker/cli/pull/2743

Legacy syntax: `ENV var value`
Current syntax: `ENV var=value`

Pl217 committed:
Format Dockerfile

This commit is the output of running command:
`yarn prettier -w .`

Pl217 committed:
Ignore revision in which Dockerfile got formatted

unocha-hpc reviewed (approved):
Checks have passed and this pull request is ready for manual review

manelcecs reviewed (approved):
All good

unocha-hpc reviewed (approved):
Checks have passed and this pull request is ready for manual review
